File: spec/code_samples/Python/wallets/get.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("GET", "/wallets?exchange=SOME_STRING_VALUE&keyId=SOME_STRING_VALUE")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: spec/code_samples/Node unirest/wallets/get.js
var unirest = require("unirest");

var req = unirest("GET", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/wallets");

req.query({
  "exchange": "SOME_STRING_VALUE",
  "keyId": "SOME_STRING_VALUE"
});


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
